SPECS-EXTENDED/libvpx/libvpx.spec

```diff
@@ -164,6 +164,8 @@ ln -sf libvpx.so.%{soversion} libvpx.so.%{somajor}.%{sominor}
 popd
 %endif
 
+ln -sf %{buildroot}%{_libdir}/libvpx.so.%{somajor} /usr/lib/libvpx.so.%{somajor}
```

Comment:
I'm guessing this is needed for the tests. Could you add a short comment of explanation, though?

Reply:
Or maybe nevermind, there are no tests. What is the reason for this link?